Vagrantfile

```diff
@@ -113,10 +113,14 @@ Vagrant.configure("2") do |config|
 	if CONF['_mode'] == "normal"
 		# Use the Chassis box we've built with the default config.
 		config.vm.box = "chassis/chassis"
-		config.vm.box_version = ">= 1.0, < 2.0"
+		if CONF['version'] >= 3.0
```

Comment:
What's the idea here - I think we don't want people to be able to mess something up by setting the conf version and getting a base box with an incompatible version of ubuntu. It needs to just be `>= 3.0, < 4.0` I think, the base box is tied to the puppet code etc in this repo as far as I'm aware.

Reply:
To extend my reasoning we're not applying the same logic to the base mode ubuntu version below, which we should do if the rest of these updates will run on xenial. Otherwise ditch the `CONF['version']` switch IMO

Reply:
Oh yeah. My bad! Of course it's based on the code updates hahaha. I'mma fix that now!